authors/apps/comments/models.py

```diff
@@ -0,0 +1,24 @@
+from django.conf import settings
```

Comment:
Thanks for the good work @wasibani-roy, however Settings is unused, you can consider refactoring to have clean code.